### PR TITLE
pages: add "how to svelte" document

### PR DESCRIPTION
**Pull request by lukeed**
Adds a Svelte + SvelteKit how-to guide for Pages

### CODE REVIEW

**Comment by benmccann** [comment]
looks good to me. only comment I have is that it might be worth mentioning the cloudflare adapter if only to say that it's meant for cloudflare workers which this tutorial doesn't cover so that if people see that adapter they won't be confused as to why it's not being used

**Comment by lukeed** [comment]
@benmccann Thanks! That will be included in the upcoming Workers How-to series. This is specifically the `pages.dev` website

File: products/pages/src/content/how-to/deploy-a-svelte-site.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+--		adapter: node(),
+++		adapter: require('@sveltejs/adapter-static')(),
+		// ... truncated ...
+	}
+};
+```
+
+<Aside>
+
+**Important:** Remember to commit and push these changes to your GitHub!
```
Comment (pngwn) [inline products/pages/src/content/how-to/deploy-a-svelte-site.md]: Should this be "GitHub repository." ?

**Comment by pngwn** [comment]
I don't have full context here but it is expected that user will know that `pages` is only for static sites currently (which isn't a constraint Svelte/ SvelteKit has, hence the potential for confusion), or is it worth adding a note about that when discussing changing the adapter?

Other than that, it looks good to me!

File: products/pages/src/content/how-to/deploy-a-svelte-site.md (hunk 1 above)
Comment (lukeed) [reply products/pages/src/content/how-to/deploy-a-svelte-site.md]: aye probably 🙃

**Comment by lukeed** [comment]
@pngwn Yes, this is the `pages.dev` website, which already covers the product details. It's static content only, and all How-To guides are leading each framework in a direction that's static-only.

**Review by kristianfreeman — APPROVED** [review]

**Pull request merged**